### PR TITLE
Spam test for send raw private transaction

### PR DESCRIPTION
**Pull request by achraf17**
### Summary

* Added a spam test of sending multiple raw private transaction between different nodes which is being signed externally
* Verify that the value stored in the simple storage contract match the randomly generated one 
* Run the new spam raw private transaction test against besu network

### CODE REVIEW

File: src/test/java/com/quorum/gauge/PrivateSmartContract.java [hunk 1]
```diff
@@ -422,4 +422,33 @@ public void sendPrivateSmartContracts(int count, QuorumNode source, String targe
...
+        }
+
+        try {
+            List deployedContracts = Observable.zip(
+                rawContractService.createNRawSimplePrivateContract(count, wallet, source, targetNodes),
+                (a) -> a
+            ).collectInto(new ArrayList<>(), (objects, deployedContract) -> objects.addAll(Arrays.asList(deployedContract))).subscribeOn(Schedulers.io()).blockingGet();
+            DataStoreFactory.getScenarioDataStore().put(store, deployedContracts);
+        } catch (Exception e) {
+            Assert.fail();
```
Comment (ricardolyn) [inline src/test/java/com/quorum/gauge/PrivateSmartContract.java:441]: we should provide more data on this, like `e.getMessage()` or other

File: src/test/java/com/quorum/gauge/PrivateSmartContract.java [hunk 2]
```diff
@@ -422,4 +422,33 @@ public void sendPrivateSmartContracts(int count, QuorumNode source, String targe
...
+        }
+    }
+
+    @Step("Verify the stored data <store> into the store to have the random value")
```
Comment (ricardolyn) [inline src/test/java/com/quorum/gauge/PrivateSmartContract.java:445]: `to have the the right value` or something similar. we know that is random, but the step checks that the deployed has the right value comparing to the used in the constructor.

File: src/specs/01_basic/private_raw_smart_contract_spam.spec [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Multiple private raw smart contracts between nodes signed externally
+
+ Tags: basic, spam
```
Comment (ricardolyn) [inline src/specs/01_basic/private_raw_smart_contract_spam.spec:3]: add `raw` tag.

Comment (achraf17) [reply src/specs/01_basic/private_raw_smart_contract_spam.spec]: added

File: src/test/java/com/quorum/gauge/PrivateSmartContract.java (hunk 2 above)
Comment (achraf17) [reply src/test/java/com/quorum/gauge/PrivateSmartContract.java]: updated

File: src/test/java/com/quorum/gauge/PrivateSmartContract.java (hunk 1 above)
Comment (achraf17) [reply src/test/java/com/quorum/gauge/PrivateSmartContract.java]: added

File: src/specs/01_basic/private_raw_smart_contract_spam.spec [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Multiple private raw smart contracts between nodes signed externally
+
+ Tags: basic, spam, raw
+
+## Send transactions from one node to others
+
+* Send "30" simple private raw smart contracts from wallet "Wallet1" in "Node1" and it's separately private for "Node2,Node3" and saved under "contratctsData1"
```
Comment (ricardolyn) [inline src/specs/01_basic/private_raw_smart_contract_spam.spec:7]: typo `contratcts`

**Review by ricardolyn — APPROVED** [review]

File: src/specs/01_basic/private_raw_smart_contract_spam.spec [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Multiple private raw smart contracts between nodes signed externally
+
+ Tags: basic, spam, raw
```
Comment (ricardolyn) [inline src/specs/01_basic/private_raw_smart_contract_spam.spec:3]: actually, we need to run this against Besu. with this, it won't work.

we need to change this line https://github.com/ConsenSys/quorum-acceptance-tests/blob/master/.github/workflows/run.yml#L125 to run this. you can change `!spam` to `(spam && raw)` and see if this tests runs as part of the Besu one

**Review by ricardolyn — CHANGES_REQUESTED** [review]

File: src/specs/01_basic/private_raw_smart_contract_spam.spec (hunk 5 above)
Comment (ricardolyn) [reply src/specs/01_basic/private_raw_smart_contract_spam.spec:3]: as this is part of the EPIC for privacy interop with Besu

Comment (achraf17) [reply src/specs/01_basic/private_raw_smart_contract_spam.spec:3]: updated

File: src/specs/01_basic/private_raw_smart_contract_spam.spec (hunk 4 above)
Comment (achraf17) [reply src/specs/01_basic/private_raw_smart_contract_spam.spec]: updated

**Review by ricardolyn — APPROVED** [review]

**Pull request merged**